lib/rules/no-jquery-methods.js

```diff
@@ -52,23 +40,14 @@ module.exports = {
       },
 
       VariableDeclarator(node) {
-        const varDecIDName = get(node, 'id.name');
-        // Get the name of what was assigned to the variable.
-        const varAssignment = get(node, 'init.callee.property.name')
-          || get(node, 'init.property.name')
-          || get(node, 'init.name')
-          || '';
-        const varAssignmentIsJquery = JQUERY_ALIASES.includes(varAssignment);
-
-        if (varAssignmentIsJquery) {
-          jQueryLocalName = varDecIDName;
-        }
+        const varName = get(node, 'id.name');
+        jQueryLocalName = isVariableAssigmentJquery(node) ? varName : null;
```

Comment:
This still only allows a single local for jQuery right?

E.g. given this snippet:

```js
import jQ from 'jquery';
const dolla = Ember.$;

dolla('.foo').focus();
jQ('.derp').focus();
```

We would only consider `dolla('.foo').focus()` incorrect (because the `const jQ = Ember.$` assignment happens and resets `jQueryLocalName`).

Reply:
Yes, I was originally only trying to solve for the `no-global-jquery` case, but it makes to fix this behavior here too